sql/backfill.go

```diff
@@ -95,22 +96,26 @@ func (p *planner) backfillBatch(b *client.Batch, tableName *parser.QualifiedName
 	}
 
 	if len(droppedColumnDescs) > 0 {
-		var updateExprs parser.UpdateExprs
-		for _, droppedColumnDesc := range droppedColumnDescs {
-			updateExprs = append(updateExprs, &parser.UpdateExpr{
-				Names: parser.QualifiedNames{&parser.QualifiedName{Base: parser.Name(droppedColumnDesc.Name)}},
-				Expr:  parser.DNull,
-			})
-		}
-
-		// Run `UPDATE <table> SET col1 = NULL, col2 = NULL, ...` to clear
-		// the data stored in the columns being dropped.
-		if _, err := p.Update(&parser.Update{
-			Table: table,
-			Exprs: updateExprs,
-		}); err != nil {
+		// Run a scan across the table using the primary key.
```

Comment:
Thanks. That wasn't obvious at all. I think the code deserves a comment so that it doesn't get accidentally reverted to using SQL for the implementation.